sbioapputils/app_runner/dev_utils.py

```diff
@@ -31,26 +32,26 @@ def _run_pycodestyle(filename):
             
 
 def validate_yaml_stages(yaml_dict, style_check = False):
-    """Validating parameters from workflow yaml"""
-    stages = yaml_dict['stages']
-    
+    """Validating parameters from workflow yaml"""    
     valid_check = True
     invalid_stage = []
     invalid_path = []
     code_errors = []
     
-    for key in stages.keys():
-
+    for key in yaml_dict['stages'].keys():
+        
         #catch common yaml formating errors and check target is a python file
-        for subkey, value in stages[key].items():
+        for subkey, value in yaml_dict['stages'][key].items():
```

Comment:
then this line will be
`for subkey, value in stageitems():`

also it is better to use more clear variable instead of `value`, cause value is very broad